src/mono/System.Private.CoreLib/src/System/GC.Mono.cs

```diff
@@ -319,7 +319,7 @@ public static TimeSpan GetTotalPauseDuration()
 
         public static System.Collections.Generic.IReadOnlyDictionary<string, object> GetConfigurationVariables()
         {
-            throw new NotImplementedException();
+            return new System.Collections.Generic.Dictionary<string, object>();
```

Comment:
```suggestion
            return System.Collections.Immutable.ImmutableDictionary<string, object>.Empty;
```

[`ImmutableDictionary<TKey, TValue>.Empty`](https://docs.microsoft.com/dotnet/api/system.collections.immutable.immutabledictionary-2.empty)

Reply:
This won't work. Corelib does not have access to ImmutableDictionary.

Reply:
Oops. Sad.